api-model/src/main/java/io/enmasse/iot/model/v1/ImageOverride.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018, EnMasse authors.
```

Comment:
That would be 2019 … unless you do time travel :wink: 

Same for all the others. Unless they are really from 2018, then it would be "2018-2019".